audio-device-client/explainer.md

```diff
@@ -0,0 +1,167 @@
+# Audio Device Client: Better and Faster Audio I/O on Web
+
+This is a proposal for a new web API, `Audio Device Client`, which funcions as
+an intermediate layer between Web Audio API and actual audio devices used by the
```

Comment:
It's not really a layer between Web Audio and audio devices. It's totally independent, right?

Reply:
In terms of the layering, ADC sits between AudioContext and UA's audio service. The ADC __CAN__ be a destination of an AudioContext, and also an AudioContext can be used standalone just like now.

The attached diagram explains this better.